commands/channel.go

```diff
@@ -116,6 +118,17 @@ Channel can be specified by team. ie. --team myTeam myChannel or by team ID.`,
 	RunE: withClient(searchChannelCmdF),
 }
 
+var MoveChannelCmd = &cobra.Command{
+	Use:   "move [team] [channels]",
+	Short: "Moves channels to the specified team",
+	Long: `Moves the provided channels to the specified team.
+Validates that all users in the channel belong to the target team. Incoming/Outgoing webhooks are moved along with the channel.
+Channels can be specified by [team]:[channel]. ie. myteam:mychannel or by channel ID.`,
+	Example: "  channel move newteam oldteam:mychannel",
+	Args:    cobra.MinimumNArgs(2),
```

Comment:
I believe this can be `cobra.ExactArgs(2)`

Reply:
This comment escaped from me, sorry for the late reply. Actually we can move several channels at a time. Therefore I used `cobra.MinimumNArgs(2)`.